### DISCUSSION TITLE
Textfield underline color

### DISCUSSION THREAD

**Discussion by gabrielporcher** [wix/react-native-ui-lib#1826] [Q&A] [answered]
Hi. I'm new at using UI libraries in RN, so sorry me if this is a stupid question.
I'm trying to change this blue/purple textfield underline color when its selected, but I just cant change it.

![rnuilib](https://user-images.githubusercontent.com/44413897/152804637-e198f8ef-f17b-475d-a596-e86265afa1d9.png)

I tried to modify:
containerStyle
fieldStyle
labelColor
and default style, but none of this works.

I couldnt find it on docs, can anyone one help me? :C

**Comment by ethanshar** [comment, marked as answer]
Hi @gabrielporcher 
That's actually a bug. 
I pushed a PR with a fix (https://github.com/wix/react-native-ui-lib/pull/1836)

The right way to set the underlineColor is to use the `fieldStyle` prop. 

The current default style comes from the `preset` prop and you couldn't override it (that's the bug)
Till this is fixed, you can override the `preset` and pass `null`, this will get you clean TextField you can customize as you wish. 
Then you'll just need to pass the `fieldStyle` with the desired style.

**Comment by gabrielporcher** [comment]
Thank you @ethanshar :smiley:
